person.rb

```diff
@@ -1,14 +1,15 @@
 require_relative 'nameable'
 
 class Person < Nameable
-  attr_accessor :id, :name, :age
+  attr_accessor :id, :name, :age, :rentals
 
   def initialize(age = nil, name = 'Unknown', parent_permission: true)
     super()
     @id = generate_id
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
   def can_use_services?
```

Comment:
- [ ] It would be nice if you could have a method that allows for adding rental in this class. It's a requirement that a method for adding rentals should be present. Kindly work on that 👍🏽